packages/client/e2e/utils.ts

```diff
@@ -2,6 +2,11 @@ import { expect } from "@playwright/test"
 import { Locator } from "playwright"
 
 export const OPENFIN_PROJECT_NAME = "openfin"
+export enum Timeout {
+  AGGRESSIVE = 5000,
```

Comment:
I would avoid words like AGGRESSIVE in general, it's a bit harsh, and for a lot of cases it is just the appropriate value so may give the wrong impression (we are not trying to catch the app out 😁) .. but we can fine tune all of this as we proceed